cli/cli_click.go

```diff
@@ -2,14 +2,31 @@ package cli
 
 import (
 	"bufio"
+	"context"
 	"fmt"
+	"net/url"
 	"os"
 
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
 	"github.com/urfave/cli/v2"
 )
 
 // Click is hidden cmd used to open some info when clicking on desktop icon
 func (c *cmd) Click(ctx *cli.Context) error {
+	if ctx.NArg() == 1 {
+		url, err := url.Parse(ctx.Args().First())
+		if err != nil {
+			return formatError(err)
+		}
+
+		if url.Scheme == "nordvpn-sl" && url.Host == "claim-online-purchase" {
+			_, err := c.client.ClaimOnlinePurchase(context.Background(), &pb.Empty{})
+			if err != nil {
```

Comment:
On successful login there is a message to let the user know `color.Green(LoginSuccess, ctx.App.Name)`, isn't needed also in this case something similar?